src/main/fc/rc_controls.h

```diff
@@ -37,12 +37,15 @@ typedef enum rc_alias {
     AUX9,
     AUX10,
     AUX11,
-    AUX12
+    AUX12,
+    COLLECTIVE = AUX3,              //woga65: for heli-like aircraft
```

Comment:
Isn't AUX2 a more common channel for collective? This matches defaults for both Spektrum and Futaba.

At least historically, futaba radios would use channel 5 for tail gain, and channel 6 (aux2) for collective.

Reply:
> Isn't AUX2 a more common channel for collective? This matches defaults for both Spektrum and Futaba.
> 
> At least historically, futaba radios would use channel 5 for tail gain, and channel 6 (aux2) for collective.

Just a matter of personal preference, will change it to whatever is agreed on. I was using CH5 for arming and CH6 to switch navigation modes from the beginning, so I opted for CH7 onwards to do additional stuff.

Btw @MrD-RC mentioned that CH5 could make some difficulties on ELRS. I cannot confirm this because I do not use ELRS yet.